doc/classes/TextEdit.xml

```diff
@@ -259,6 +268,7 @@
 			<argument index="0" name="option" type="int">
 			</argument>
 			<description>
+				Triggers a right click menu action by the specified index. See [enum MenuItems] for a list of available indexes.
```

Comment:
We seem to use both "indexes" and "indices" throughout the classref, we should probably decide which one is correct and enforce its use. CC @cbscribe @NathanLovato